### PR TITLE
WIP - Updating Drupal 8 migration information

### PR DESCRIPTION
**Pull request by stevector**
Closes https://github.com/pantheon-systems/documentation/issues/1386

## Effect
PR includes the following changes:
- Increases detail of information for Drupal 8 migrations.

## Remaining Work
- [x] Editorial review
- [x] Add information switching DNS from a Drupal 6 or 7 site live on Pantheon to the new Drupal 8 site.

### CODE REVIEW

**Comment by stevector** [comment]
@rachelwhitton, do you know if we have info specifically on switching a domain from one Pantheon site to another Pantheon site?

**Comment by rachelwhitton** [comment]
@stevector [Switching DNS From One Pantheon Site to Another](https://pantheon.io/docs/switching-dns/)

**Comment by stevector** [comment]
Thanks @rachelwhitton, can you review?

File: source/_docs/drupal-updates.md [hunk 1]
```diff
@@ -27,20 +28,58 @@ If you are not having much luck with update.php, consider setting up the new sit
...
+terminus drush my-drupal-8-site.dev -- migrate-upgrade --legacy-db-key=drupal_7 --configure-only --legacy-root=http://drupal7.example.com
+```
+This command configures (but does not run) the migrations from Drupal 6 or 7 to Drupal 8. In this example, the Drupal 8 site is named `my-drupal-8-site` and the command is running on the `dev` environment. The `--legacy-db-key` parameter indicates how to get the login credentials to the source Drupal 6 or 7 database. In our example, we use the [Terminus secrets](https://github.com/pantheon-systems/terminus-secrets-plugin) plugin to supply the connection info. [See our blog post for more information on how this flag is used](https://pantheon.io/blog/running-drupal-8-data-migrations-pantheon-through-drush). The `--legacy-root` flag lets Drupal 8 know from where it can grab images and other uploaded media assets.
+
+The following command generates a report on how many items have been imported by each migration:
+```
+terminus drush my-drupal-8-site.dev -- migrate-status
+```
+
+The following command runs the migration configured via `drush migrate-upgrade`:
```
Comment (rachelwhitton) [inline source/_docs/drupal-updates.md]: Is this accurate @stevector? I may have misunderstood the original context.

**Comment by rachelwhitton** [comment]
+1 One comment, but otherwise looks great! Thanks @stevector

File: source/_docs/drupal-updates.md (hunk 1 above)
Comment (stevector) [inline source/_docs/drupal-updates.md]: Yes, `drush migrate-import` is what runs the migration. `drush migrate-upgrade` can both configure and run migrations. Above I include the `--configure-only` flag. I could do that here too so this line would read as 

> The following command runs the migration configured via `drush migrate-upgrade --configure-only`:

File: source/_docs/drupal-updates.md [hunk 2]
```diff
@@ -27,20 +28,58 @@ If you are not having much luck with update.php, consider setting up the new sit
...
+This command configures (but does not run) the migrations from Drupal 6 or 7 to Drupal 8. In this example, the Drupal 8 site is named `my-drupal-8-site` and the command is running on the `dev` environment. The `--legacy-db-key` parameter indicates how to get the login credentials to the source Drupal 6 or 7 database. In our example, we use the [Terminus secrets](https://github.com/pantheon-systems/terminus-secrets-plugin) plugin to supply the connection info. [See our blog post for more information on how this flag is used](https://pantheon.io/blog/running-drupal-8-data-migrations-pantheon-through-drush). The `--legacy-root` flag lets Drupal 8 know from where it can grab images and other uploaded media assets.
+
+The following command generates a report on how many items have been imported by each migration:
+```
+terminus drush my-drupal-8-site.dev -- migrate-status
+```
+
+The following command runs the migration configured via `drush migrate-upgrade --configure-only`:
+```
+terminus drush $SITE_ENV -- migrate-import --all
```
Comment (rachelwhitton) [inline source/_docs/drupal-updates.md]: Is there a reason we're using `$SITE_ENV` instead of `<site>.<env>`?

**Pull request merged**